features/products.feature

```diff
@@ -70,4 +71,15 @@ Scenario: Delete a Product
     And I press the "Delete" button
     Then I should see the message "Product has been Deleted!"
     When I press the "Retrieve" button
-    Then I should see the message "404 Not Found: Product with id"
\ No newline at end of file
+    Then I should see the message "404 Not Found: Product with id"
+
+Scenario: List all Products
+    When I visit the "Home Page"
+    And I press the "Clear" button
```

Comment:
@luoyantao99 if the Category and Available are not selected by default, that we do not need to clear  before list all product